### ISSUE TITLE
ESPN Addon Widevine error & loop

### ISSUE THREAD

**Issue by ncsufan8628** [matthuisman/slyguy.addons#534]
When accessing content via espn I get a widevine error about not being on the latest version despite having it installed.

If I dismiss the error some content will play, ~15 secs that just continues to loop.  Logs attached.

[kodi.log](https://github.com/matthuisman/slyguy.addons/files/12329637/kodi.log)

**Comment by matthuisman** [comment]
[script.module.slyguy -> v0.70.7](https://github.com/matthuisman/slyguy.addons/commit/6047fda3b0bbb0a041004758d16e20926b9db2f4) should hopefully fix. please test and let me know

**Comment by ncsufan8628** [comment]
so no more Widevine error, but it still seems to just loop about 15s of whatever I play.

**Comment by matthuisman** [comment]
are you using TV provider login or ESPN+ login?
Do all channels loop?

Have you tried Kodi Omega by any chance?
It has quite a few inputstream.adaptive improvements that may solve the issue

**Comment by ncsufan8628** [comment]
I am using Kodi Nexus (I think) via OSMC on a RPi4, 4gb.  

TV provider login is being used.  

Not sure what you mean by do all channels loop, but my MLBTV addon seems to work fine as does your IPTV manager.  I have also noticed no difference between which item I pick to play within the ESPN addon itself.   I've tried on my other RPi4 as well as a fresh install of Kodi on Mac and have the same problem w/ the addon.

**Comment by matthuisman** [comment]
Pretty sure you will need Kodi Omega (newer inputstream adaptive) to fix this issue.
This is due to <location> tag support not working correctly in older inputstream adaptives.

There is still an issue with Omega though when used in conjunction with slyguy quality select.
But Kodi 21.0 Omega + slyguy bypass quality = plays OK

**Comment by ncsufan8628** [comment]
Oh let me try.  I def didn't have bypass enabled.

On Tue, Aug 29, 2023 at 10:46 PM Matt Huisman  <

**Comment by ncsufan8628** [comment]
yep working with the latest nightly of Omega.  

assuming there will be no hope then for current Nexus and earlier releases and we will just have to wait for omega to hit those devices?

**Comment by matthuisman** [comment]
Yes. The location support rework won't be backported to earlier inputstream
versions unfortunately

**Comment by matthuisman** [comment]
[script.module.slyguy -> v0.71.2](https://github.com/matthuisman/slyguy.addons/commit/bc008c1c71cd6448025b313ec4e81de08d23d442) should allow quality control to work with ESPN + Kodi Omega now.

Please test and let me know :)

**Comment by ncsufan8628** [comment]
working here w/ quality select enabled!
